src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Semantic/TestFiles/DefaultRazorSemanticTokenInfoServiceTest/GetSemanticTokens_CSharpBlock_HTML.semantic.txt

```diff
@@ -1,18 +1,18 @@
 //line,characterPos,length,tokenType,modifier
-0 0 1 84 0 //razorTransition
```

Comment:
Storing the TokenType as numbers may no longer be a good idea now that the order is programatic.

Reply:
I changed the test format so that instead of
```
0 0 1 84 0 //razorTransition
```
we get something like
```
0 0 1 razorTransition 0
```
Let me know if that looks better to you!

Reply:
That looks great!